libs/database/src/lib/crud/users.ts

```diff
@@ -34,6 +34,10 @@ export const getUserWithCredentials = (filter: Filter<User>) => {
   return db.collection<User>('users').findOne(filter);
 };
 
+export const getUsersWithCredentials = (filter: Filter<User>) => {
```

Comment:
How is this "with credentials"? It looks like a regular users call

Reply:
Users usually return with credentials, it is when we want to return a safe user to the backend the we omit the "with credentials" and do some more logic (this follows similar functions in the same file)